doc/source/whatsnew/v0.21.0.txt

```diff
@@ -35,6 +35,7 @@ Other Enhancements
 - ``RangeIndex.append`` now returns a ``RangeIndex`` object when possible (:issue:`16212`)
 - :func:`to_pickle` has gained a protocol parameter (:issue:`16252`). By default, this parameter is set to `HIGHEST_PROTOCOL <https://docs.python.org/3/library/pickle.html#data-stream-format>`__
 - :func:`api.types.infer_dtype` now infers decimals. (:issue: `15690`)
+- :func:`clip` now supports inplace trimming. (:issue: `15388`)
```

Comment:
no, ``DataFrame.clip()`` and ``Series.cip()`` have gained an ``inplace`` argument.